dv/uvm/core_ibex/scripts/compile-tb.py

```diff
@@ -35,6 +37,32 @@ def main() -> int:
 
     os.makedirs(args.output, exist_ok=True)
 
+    # Setup environment variables for the tb to link against the compiled ISS
+    if args.en_cosim:
+        _spike_iss_pc = 'riscv-riscv riscv-disasm riscv-fdt'
+
+        if not subprocess.run(shlex.split(f'pkg-config --exists {_spike_iss_pc}'),
+                              check=True):
+            raise EnvironmentError(f'Failed to find "{_spike_iss_pc}" pkg-config\
+            packages. Did you set the PKG_CONFIG_PATH correctly?')
+
+        def _get_iss_pkgconfig_flags(cmd, iss_pc):
+            _flag = subprocess.check_output(
+                args=shlex.split(f'pkg-config {cmd} {iss_pc}'),
+                universal_newlines=True)
+            if args.simulator == 'xlm':
+                return _flag.replace('-Wl,', '')  # Don't ask
```

Comment:
I like the comment :-) But we should probably say something a little more informative for the next poor soul that has to make sense of this!